src/broker/cas_execute.c

```diff
@@ -2708,6 +2710,10 @@ ux_execute_array (T_SRV_HANDLE * srv_handle, int argc, void **argv, T_NET_BUF *
 
       if (res_count < 0)
 	{
+	  if (res_count == ER_QPROC_INVALID_XASLNODE && retried_query_num != num_query)
+	    {
+	      goto exec_retry_xasl_error;
+	    }
```

Comment:
My suggestion (it is trivial and do as you like):
1. remove goto statement & copy goto block here (2816 ~ 2824) like this:
```
 if (res_count == ER_QPROC_INVALID_XASLNODE && retried_query_num != num_query)
  {
  err_code = recompile_statement (srv_handle);
  if (err_code < 0)
    {
      goto exec_db_error;
    }
  session = (DB_SESSION *) srv_handle->session;
  retried_query_num = num_query;
  num_query--;
  continue;
}
```

2. Remove `continue `of line# 2813, and 'exec_retry_xasl_error:' of line# 2815